### PR TITLE
Deadce11 test coverage

### PR DESCRIPTION
**Pull request by MrDeadCe11**
added a script to generate an html coverage report and an update to the Readme.  A small pr to test the waters.

### CODE REVIEW

**Comment by pi0neerpat** [comment]
Closes #337

**Review by daopunk — CHANGES_REQUESTED** [review]
can you run `yarn lint:fix & yarn lint:sol` so linter passes in github actions

**Comment by daopunk** [comment]
also, what did the report say for coverage?

**Comment by MrDeadCe11** [comment]
pushed up lints and coverage report folder and report.

**Review by daopunk — APPROVED** [review]
Great!

File: test/testlocal/nft/anvil/deployment/AnvilContracts.t.sol [hunk 1]
```diff
@@ -5,6 +5,7 @@ abstract contract AnvilContracts {
   address public ChainlinkRelayerFactory_Address = 0x5FbDB2315678afecb367f032d93F642f64180aa3;
   address public DenominatedOracleFactory_Address = 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512;
   address public DelayedOracleFactory_Address = 0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0;
+  address public InternalCallsExtension_Address = 0x0DCd1Bf9A1b36cE34237eEaFef220932846BCD82;
```
Comment (pi0neerpat) [inline test/testlocal/nft/anvil/deployment/AnvilContracts.t.sol:8]: Whats the purpose of this line?

**Review by pi0neerpat — APPROVED** [review]
Thanks! After my approval, you are responsible for merging all PRs when you decide you are ready.

File: test/testlocal/nft/anvil/deployment/AnvilContracts.t.sol (hunk 1 above)
Comment (MrDeadCe11) [reply test/testlocal/nft/anvil/deployment/AnvilContracts.t.sol:8]: It was added when I deployed the contracts on anvil as part of the InternalCallsWatcher.sol contract.  Seems like it should have been there before?

**Pull request merged**